src/controller/java/CHIPDeviceController-JNI.cpp

```diff
@@ -852,6 +854,90 @@ JNI_METHOD(void, readPath)
     callback->mReadClient = readClient;
 }
 
+JNI_METHOD(void, subscribeToEventPath)
+(JNIEnv * env, jobject self, jlong handle, jlong callbackHandle, jlong devicePtr, jobject eventPathList, jint minInterval,
+ jint maxInterval)
```

Comment:
you also need to add isFabricFilter boolean for fabric scoped event in Event read/subscribe API and KeepSubscriptions boolean in Event Subscribe.

you may consider to have the ReadRequestParam structure to cover this.

Generally the API should support both event path and attribute path in same read/subscribe request.

Reply:
I added parameter in this function.